main_app/igdb_api.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+import requests
+from urllib.parse import urlencode, quote_plus
+
+from django.conf import settings
+
+
+class IGDB:
+    def __init__(self):
+        self.__last_headers_x_count = 1
+        self.__api_key = settings.IGDB_API_KEY
+        self.__api_url = 'https://api-v3.igdb.com/'
+        self.__headers = {
+            'user-key': self.__api_key,
+            'Accept': 'application/json'
+        }
+        self.__limit = 6
```

Comment:
let's make it dynamic.
```
def __init__(self, limit):
    self.__limit = 6
```
In this case you will be able easily change number of items per page `client = IGDB(limit=10)`